<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradienBoxtItemFrame</class>
 <widget class="QFrame" name="GradienBoxtItemFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>60</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_del">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="text">
    <string>复制文本</string>
   </property>
  </action>
  <action name="action_rename">
   <property name="text">
    <string>重命名</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
